validator/validator.go

```diff
@@ -46,6 +46,21 @@ func Cron(value interface{}, _ cty.Path) diag.Diagnostics {
 	return diags
 }
 
+func CronLength(value interface{}, _ cty.Path) diag.Diagnostics {
+	var diags diag.Diagnostics
+	cron := regexp.MustCompile("[^\\s]+").FindAllString(value.(string), -1)
+
+	if len(cron) > 7 || len(cron) < 6 {
```

Comment:
Not a blocker, my preference is to write this like I'd read it. e.g. `if len(cron) < 6 || len(cron) > 7`

Reply:
okay :)